server/src/services/album.service.ts

```diff
@@ -262,11 +258,20 @@ export class AlbumService {
       await this.access.requirePermission(auth, Permission.ALBUM_SHARE, id);
     }
 
-    await this.albumRepository.update({
-      id: album.id,
-      updatedAt: new Date(),
-      sharedUsers: album.sharedUsers.filter((user) => user.id !== userId),
-    });
+    await this.albumUserRepository.delete({ albumId: id, userId });
+  }
+
+  async updateAlbumUser(auth: AuthDto, id: string, userId: string, dto: Partial<AlbumUserEntity>): Promise<void> {
+    await this.access.requirePermission(auth, Permission.ALBUM_SHARE, id);
+
+    const album = await this.findOrFail(id, { withAssets: false });
```

Comment:
This is probably fine for now, but since we have an entity for this intersection table now, we'll eventually want to transition to a `getUser` repository method that gets a user by albumId and userId. Rather than loading every single user and filtering through them.

Reply:
Should I open a ticket? How are these tracked?

Reply:
You can add a comment in the code here if you want and/or create a new issue on github.